library/systemd/test/systemd_service_test.rb

```diff
@@ -276,26 +276,32 @@ module Yast
 
     describe "#enable" do
       subject(:service) { SystemdService.find("cups") }
+      let(:socket) { double("socket") }
 
       before do
         stub_services(service: "cups")
+        allow(service).to receive(:socket).and_return(socket)
       end
 
-      it "sets start_mode to :boot" do
-        expect(service).to receive(:start_mode=).with(:boot)
+      it "enables the service to start on boot" do
+        expect(service).to receive(:enable_service)
+        expect(socket).to receive(:disable)
```

Comment:
hmm, that in fact realize me that we are not 100% backward compatible, so maybe create separate `it` state for this side effect. As previously service.enable does not disable socket. So we should document it.